eventually/amazons3/views.py

```diff
@@ -94,6 +95,6 @@ def delete(self, request):
             BOTO_S3.Object(settings.AWS_STORAGE_BUCKET_NAME,
                            image_to_delete.get("image_key")).load()
         except ClientError:
-            return HttpResponse(status=200)
+            return RESPONSE_200_DELETED
```

Comment:
does not this response has to be inside `try` block?

Reply:
Yes, it queries Amazon for the picture. 
If it has been deleted, we get an exception. Otherwise we get the picture.